rules-topics/01_introduction.md

```diff
@@ -62,6 +62,56 @@ Welcome to Chronicles. Whether you've been LARPing for years, or have never seen
+
+If you have a character in another Larp System please see our Transfer Policy if you want to transfer your character to Chronicles.
+
+### Secondary Characters 
+Once you hit 200 Skill Points you can create a secondary character.  This character will always have 75 percent of your primary character’s skill points.  If you play a secondary character, then all gained SP will be applied to the primary character.  Strain is tracked separately on each character.
+
+If you wish to create an additional character beyond a secondary character you can.  All characters beyond the second character will have 50 percent of the primary character’s Skill Points.  All gained SP will be applied to the primary character.  Strain is tracked separately on each character.
+
+### Identify Ability
+Allows the character to Identify an item, a group of items, or target after 1 minute of concentration and continuous contact. This will make the nature and workings of enchantments on an item/creature known to the character, at the discretion of plot. There may be some properties that still remain hidden to the character.
+
+Status that require Identify:
+|Identify Needed|
+|:---:|
+|Animate|
+|Curse|
+|Curse of Transformation|
+|Infect|
+|Spirit Bottle|
+
+### Floating Reset
+Some skills or abilities are listed as "Daily".  These skills and abilities are renewed between 4:00 and 6:00 PM.  If you are on a module between that timeframe you must attempt the reset immediately after a mod.  Daily skills and abilities require 10 minutes of concentration/Rest to refresh.
+
+### How Does Combat Work
+
+Chronicles is a boffer combat Larp that focuses as much on roleplay and interaction between PCs and NPCs as physical combat.  We emphasize light combat, and a general rule is if you swing enough for it to hurt someone you are probably swinging too hard.  The goal of combat is to be safe and while accidents may happen you are more likely to twist an ancle on uneven ground than hurt yourself during combat.  More details can be found in the Combat section of the rulebook.
+
+### Plot Members and Marshals
+
+Plot Members and Marshals are officials of Chronicles.  They will give rulings, run a mod, observe a ritual, and other OOG tasks that are needed throughout a market fair.  If you have a question on a rule or how something works you will need to ask the appropriate marshal.  
+
+Plot Member – A member of the group of people who work together to keep the game moving, adjudicate rules, run mods and many other roles essential to the game.  In table tops terms they are essentially the game master or dungeon master.
+
+Mod Marshal – A person who runs a mod for a group for PCs.  They will set the stage giving descriptions, locations, etc. Should any rule questions or disputes come up, the Mod Martial has final say.  During a mod a Martial rule is final, if you feel they were incorrect it can be adjudicated later when it does not disrupt the flow of the game.
```

Comment:
"Martial" appears to be typo'd/autocorrected in here twice.

Reply:
AutoCorrect...yes that did it....
Fixed